src/modules/atommodel/temporalproofs/RadixTemporalProof.ts

```diff
@@ -41,9 +41,9 @@ export class RadixTemporalProof extends RadixSerializableObject {
 
     @includeJSON
     @includeDSON
-    object: RadixHash
+    public object: RadixHash
```

Comment:
I think this class could go?  We don't have temporal proofs any more.

Reply:
deleted